test/io/rkn/conformity_test.clj

```diff
@@ -50,6 +63,11 @@
         (is (has-attribute? (db conn) :test/attribute3))
         (is (empty? (ensure-conforms conn sample-norms-map1)))))
 
+    (testing "can add db/unique after an avet index add"
```

Comment:
This test does actually pass on master. I assume either due to some different behaviour with the memory database, or some shortcut where datomic realizes there are no datoms to index for this attribute, so the index addition is immediate.